packages/dds/tree/src/simple-tree/treeApi.ts

```diff
@@ -164,6 +172,19 @@ export const treeNodeApi: TreeNodeApi = {
 			T
 		>;
 	},
+	shortID(node: TreeNode): number | undefined {
+		const flexNode = getFlexNode(node);
+		for (const field of flexNode.boxedIterator()) {
+			if (field.schema.kind === FieldKinds.identifier) {
+				for (const child of field.boxedIterator()) {
```

Comment:
I think Craig added a `field.boxedAt()` recently - you can probably use that instead of doing an inner iteration here.

Reply:
Updated to use `field.boxedAt()` instead of iterating